### PR TITLE
qweb: solve t-field integer issue while render on frontend.

### PR DESCRIPTION
**Pull request by apa-odoo**
Task: https://www.odoo.com/web#id=10730&view_type=form&model=project.task&action=333&active_id=131
Pad: https://pad.odoo.com/p/r.CCvjCUjYWuJdpnIw

### CODE REVIEW

File: addons/website/static/src/js/website.editor.js [hunk 1]
```diff
@@ -1097,7 +1097,8 @@ var EditorBar = Widget.extend({
                         placement: 'auto top',
                     })
                     // Force-show popovers so users will notice them.
-                    .popover('show');
+                    .popover('show')
+                    .attr("contentEditable", true);
```
Comment (Gorash) [inline addons/website/static/src/js/website.editor.js]: Why ?

Comment (apa-odoo) [inline addons/website/static/src/js/website.editor.js]: @Gorash suppose there is failure in integer field (Like., Integer out of range, invalid literal for int()), then it's not editable after popup show.

Comment (Gorash) [inline addons/website/static/src/js/website.editor.js]: A popover with contentEditable ?...
Remove this change, if the 'range' problem is appear we will fix later.
Thanks

Comment (apa-odoo) [inline addons/website/static/src/js/website.editor.js]: Ok I remove it. Now, in master problem is solve. Thanks

**Pull request closed without merging**